Richard

richardholst opened the pull request:
I have updated the install script to update the yaml config file with the download location for the currently logged on user

richardholst committed:
no message

richardholst committed:
Merge branch 'master' of https://github.com/guardian/autopull into Richard

* 'master' of https://github.com/guardian/autopull:
  updated the download location with powershell variable so it downloads to the logged on users download folder

richardholst committed:
added an automated application powershell Install script for Autopull

richardholst committed:
updated downlaod location with logged on user

fredex42 commented on AutoPull_install.ps1:
```diff
@@ -0,0 +1,11 @@
+# make dir  
+mkdir "C:\Program Files (x86)\autopull\"
```
will this error if the directory already exists?

richardholst replied:
yes it will I will add a -force option so it does not error  for now ..then add  more eror checking later

fredex42 reviewed (changes requested):
please can you include the existing powershell lines to register the protocol handler in the install script?

thanks a lot

richardholst committed:
removed install_handler.ps1 script and added the code to the AutoPull_Install.ps1

Updated the AutoPull_Install.ps1 to force overwite of files if they already on the sytem - in the case of reinstalll

fredex42 reviewed (approved):
:+1: 

Much appreciated!
